build-lble/testsuite/lb_leesedwards.py

```diff
@@ -0,0 +1,162 @@
+#!/usr/bin/env python
+
+from __future__ import print_function
+
+import espressomd as md
+from espressomd import lb
+import numpy as np
+import unittest as ut
+from tests_common import abspath
+
+tol = 1.0e-5
+
+# Integration parameters
+time_step = 1.0
+total_time = 1000.0
+
+# LB parameter for offset and particle coupling test
+eta = 1.0
+rho = 1.0
+nu = eta / rho
+agrid = 1.0
+fric = 1.0
+lb1 = md.lb.LBFluidGPU(agrid=agrid, dens=rho, visc=eta, fric=fric, tau=time_step)
+
+# LB parameter for test against Navier-Stokes equation
+eta = 0.27
+rho = 0.97
+nu = eta / rho
+v = 0.0087
+k_max = 100
+agrid = 1.0
+fric = 1.1
+lb2 = md.lb.LBFluidGPU(agrid=agrid, dens=rho, visc=eta, fric=fric, tau=time_step)
```

Comment:
the gpu version is a feature that may not be compiled in, so you have to guard those statements that refer to the LBFluidGPU instance. Furthermore it's easier to set the system up in the class and only use a single testclass.